thorlcr/msort/tsorts1.cpp

```diff
@@ -69,10 +69,10 @@ class CMergeReadStream : public CSimpleInterface, public IRowStream
 #if defined(_USE_OPENSSL)
         if (secureContextClient)
         {
+            Owned<ISecureSocket> ssock = secureContextClient->createSecureSocket(socket.getClear());
             int tlsTraceLevel = SSLogMin;
-            if (sortTraceLevel)
+            if (sortTraceLevel > DebugMsgThreshold)
```

Comment:
Should this be testing if > MyVerboseLevel (which is higher than the default for now), to suppress extra tracing?